src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -453,7 +449,7 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
   const [deleteValidation, setDeleteValidation] = useState('');
   const handleDeleteClicked = (event) => {
     event.preventDefault();
-    onClose(); // Close the settings modal.
+    //onClose(); // Close the settings modal.
```

Comment:
Should this still be commented out? If so we can remove this line?

Reply:
good catch, we can remove